aldryn_events/tests/frontend/integration/pages/alteli.js

```diff
@@ -0,0 +1,166 @@
+// ALDRYN TESTS LIBRARY
+// ############################################################################
+'use strict';
+/* global expect, browser, jasmine */
+/* jshint browser: true */
+/* jshint shadow: true */
+
+var aldrynTestsLibrary = {
+    // utility to set browser window size
+    setWindowSize: function (width, height) {
+        browser.driver.manage().window().setSize(
+            width,
+            height
+        );
+    },
+
+    // utility to pre-create folder with 0777 permissions
+    preCreateFolder: function (folderName) {
+        var fs = require('fs');
+        var path = require('path');
+
+        var mkdirSync = function (path, mode) {
+            try {
+                fs.mkdirSync(path, mode);
+            } catch (e) {
+                if (e.code !== 'EEXIST') {
+                    throw e;
+                }
+            }
+        };
+        mkdirSync(path.join(folderName), 511);
+    },
+
+    // utility to automatically store a screenshot at the end of each test
+    storeScreenshot: function (screenShotDirectoryPath) {
+        // screenshots creation
+        var fs = require('fs');
+        var screenShotDirectory = screenShotDirectoryPath;
+        var Utils = {
+            /**
+             * @name writeScreenShot
+             * @description Write a screenshot string to file.
+             * @param {String} data The base64-encoded string to write to file
+             * @param {String} filename The name of the file to create (do not
+             * specify directory)
+             */
+            writeScreenShot: function (data, filename) {
+                var stream = fs.createWriteStream(
+                    screenShotDirectory + filename
+                );
+
+                stream.write(new Buffer(data, 'base64'));
+                stream.end();
+            }
+        };
+
+        jasmine.getEnv().addReporter({
+            specDone: function (result) {
+                browser.getCapabilities().then(function (capabilities) {
+                    browser.takeScreenshot().then(function (png) {
+                        var browserName = capabilities.caps_.browserName;
+                        var filename =
+                        browserName + '-' +
+                        result.status.toUpperCase() + '-' +
+                        result.fullName.replace(/ /g, '_') + '.png';
+
+                        Utils.writeScreenShot(png, filename);
+                    });
+                });
+            }
+        });
+    },
+
+    // utility to test expectation an element is missing
+    expectByCssToBeAbsent: function (element) {
+        element.isPresent().then(function (present) {
```

Comment:
no. this method (`expectByCssToBeAbsent`) currently returns undefined. but inside there is an asynchronous action that returns a promise. it might make sense to return that promise from a method so you can do `x.expectByCssToBeAbsent(element).then(function () { /* yadda yadda */ }`